package.json

```diff
@@ -108,119 +107,12 @@
-		"@automattic/request-external-access": "1.0.0",
-		"@automattic/social-previews": "1.1.0",
-		"@automattic/viewport": "1.0.0",
-		"@babel/core": "7.12.9",
-		"@babel/preset-env": "7.12.7",
-		"@babel/register": "7.12.1",
-		"@wordpress/annotations": "1.12.2",
-		"@wordpress/base-styles": "2.0.1",
-		"@wordpress/browserslist-config": "2.6.0",
-		"@wordpress/compose": "3.11.0",
-		"@wordpress/data": "4.14.2",
-		"@wordpress/date": "3.11.1",
-		"@wordpress/element": "2.18.0",
-		"@wordpress/i18n": "3.9.0",
-		"@wordpress/icons": "2.7.0",
-		"@wordpress/url": "2.11.0",
-		"axios": "0.21.0",
-		"babel-jest": "26.6.3",
-		"babel-loader": "8.2.2",
-		"bounding-client-rect": "1.0.5",
-		"cache": "2.1.0",
-		"classnames": "2.2.6",
-		"click-outside": "2.0.2",
-		"clipboard": "2.0.6",
-		"component-uid": "0.0.2",
-		"cookie": "0.4.1",
-		"copy-webpack-plugin": "5.1.2",
-		"create-react-class": "15.6.3",
-		"cross-env": "7.0.2",
-		"debug": "4.3.1",
-		"email-validator": "2.0.4",
-		"fancy-log": "1.3.3",
-		"fast-json-stable-stringify": "2.1.0",
-		"focus-trap": "6.2.1",
-		"gridicons": "3.3.1",
-		"gulp": "4.0.2",
-		"gulp-append-prepend": "1.0.8",
-		"gulp-autoprefixer": "7.0.1",
-		"gulp-clean-css": "4.3.0",
-		"gulp-concat": "2.6.1",
-		"gulp-if": "3.0.0",
-		"gulp-minify": "3.1.0",
-		"gulp-modify-css-urls": "2.0.0",
-		"gulp-rename": "2.0.0",
-		"gulp-rtlcss": "1.4.1",
-		"gulp-sass": "4.1.0",
-		"gulp-sourcemaps": "3.0.0",
-		"jsdom": "16.4.0",
-		"jsdom-global": "3.0.2",
-		"json-loader": "0.5.7",
-		"mapbox-gl": "1.13.0",
-		"markdown-it": "12.0.2",
-		"node-sass": "4.14.1",
-		"page": "1.7.1",
-		"photon": "2.1.0",
-		"plugin-error": "1.0.1",
-		"postcss-custom-properties": "10.0.0",
-		"preact": "10.5.7",
-		"progress-event": "1.0.0",
-		"prop-types": "15.7.2",
-		"q-flat": "1.0.7",
-		"qss": "2.0.3",
-		"react-pure-render": "1.0.2",
-		"react-redux": "6.0.1",
-		"react-router-dom": "5.2.0",
-		"react-tagsinput": "3.19.0",
-		"redux": "4.0.5",
-		"redux-thunk": "2.3.0",
-		"refx": "3.1.1",
-		"rememo": "3.0.0",
-		"resize-observer-polyfill": "1.5.1",
-		"static-site-generator-webpack-plugin": "3.4.2",
-		"strip": "3.0.0",
-		"swiper": "6.3.5",
-		"tinycolor2": "1.4.2",
-		"uglify-save-license": "0.4.1",
-		"url-polyfill": "1.1.12",
-		"uuid": "8.3.1",
-		"webpack": "4.44.2",
-		"webpack-cli": "4.2.0"
+		"**/terser-webpack-plugin/serialize-javascript": "^5.0.0",
+		"prettier": "npm:wp-prettier@2.0.5"
 	},
 	"devDependencies": {
-		"@automattic/color-studio": "2.3.1",
-		"@babel/plugin-transform-runtime": "7.12.1",
-		"@size-limit/preset-app": "4.9.0",
-		"@slack/web-api": "5.14.0",
-		"@testing-library/jest-dom": "5.11.6",
-		"@testing-library/preact": "2.0.0",
-		"@wordpress/components": "9.2.6",
-		"@wordpress/core-data": "2.12.3",
-		"@wordpress/dependency-extraction-webpack-plugin": "2.4.0",
-		"@wordpress/e2e-test-utils": "4.3.1",
-		"@wordpress/editor": "9.12.7",
-		"@wordpress/env": "1.6.0",
-		"@wordpress/eslint-plugin": "4.0.0",
```

Comment:
@wordpress/eslint-plugin is still needed.
```$ yarn lint
yarn run v1.22.10
$ eslint --ext .js,.jsx . _inc extensions modules

Oops! Something went wrong! :(

ESLint: 7.14.0

ESLint couldn't find the plugin "@wordpress/eslint-plugin".
```

Reply:
Ouch. Not sure why it was working for me. I guess there were some leftovers in `node_modules`